core/amber/src/main/resources/python_udf/topic_modeling_mallet_trainer.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+# Use gensim 3.8.3
```

Comment:
we might need to find a better way to configure ENV but for now we can keep it like this.